src/Actions/Actions.ts

```diff
@@ -65,29 +65,30 @@ export default class Actions implements IActions {
       this.ledger.accountClaiming(receipt, txRequest, opts, from);
 
       switch (status) {
-        case TxSendErrors.OK:
+        case TxSendStatus.OK:
           this.cache.get(txRequest.address).claimedBy = from;
-          return ClaimStatus.SUCCESS;
-        case TxSendErrors.WALLET_BUSY:
-          return ClaimStatus.ACCOUNT_BUSY;
-        case TxSendErrors.IN_PROGRESS:
-          return ClaimStatus.IN_PROGRESS;
-        case TxSendErrors.MINED_IN_UNCLE:
-          return ClaimStatus.MINED_IN_UNCLE;
+          return TxSendStatus.STATUS(context, TxSendStatus.SUCCESS);
+        case TxSendStatus.WALLET_BUSY:
+          return TxSendStatus.STATUS(context, TxSendStatus.BUSY);
+        case TxSendStatus.IN_PROGRESS:
+          return TxSendStatus.STATUS(context, TxSendStatus.PROGRESS);
+        case TxSendStatus.MINED_IN_UNCLE:
+          return TxSendStatus.STATUS(context, TxSendStatus.MINED);
```

Comment:
```suggestion
          return TxSendStatus.STATUS(context, status);
```

and remove the whole `switch`? Looks like TxSendStatus.MINED_IN_UNCLE and TxSendStatus.MINED are redundant, and other pairs above as well.

Reply:
@kosecki123  the status and return type are two different string messages so i can't just remove switch and return status and i didn't change this part, it was like this from before only but if you want me to change it can you be more specific about what to change.
Also MINED_IN_UNCLE and MINED are two different strings that's why i was passing MINED as a string message instead of making a similar field in enum.

Reply:
> @kosecki123 the status and return type are two different string messages so i can't just remove switch and return status and i didn't change this part, it was like this from before only but if you want me to change it can you be more specific about what to change.
> Also MINED_IN_UNCLE and MINED are two different strings that's why i was passing MINED as a string message instead of making a similar field in enum.

Then maybe split string messages and enum. The current code does not look right.